File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/OApi/src/main/java/com/udid/oapi/data/PABillRegisterData.java
package com.udid.oapi.data;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import lombok.Builder;
import lombok.Getter;
import lombok.NonNull;
import lombok.Setter;
import lombok.ToString;

/**
 * Create by 김진원
 */
@Getter
@Setter
@ToString
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public class PABillRegisterData {

    /**
     * 판매자 회원 아이디
     * 필수
     */
    @NonNull
    String userid;

    /**
     * 카드번호
     * 필수
     */
    @NonNull
    String cardNo;

    /**
     * 카드 유효기간(월)
     * 필수
     */
    @NonNull
    String expMonth;

    /**
     * 카드 유효기간(년)
     * 필수
     */
    @NonNull
    String expYear;

    /**
     * 구매자확인 개인:생년월일(YYMMDD) 6자리 , 사업자번호
     * 필수
     */
    @NonNull
    String buyerAuthNo;

    /**
     * 카드 비밀번호 앞 두자리
     * 필수
     */
    @NonNull
    String cardPw;

    /**
     * 구매자 전화번호
     * 필수
     */
    @NonNull
    String buyerPhone;

    /**
     * 구매자 성함
     * 필수
     */
    @NonNull
    String buyerName;

    /**
     * 구매자 아이디
     */
    String buyerId;

    private static PABillRegisterDataBuilder builder() {
        return new PABillRegisterDataBuilder();
    }

    /**
     * 필수 값
     * @param userid
     * @param cardNo
     * @param expMonth
     * @param expYear
     * @param buyerAuthNo
     * @param cardPw
     * @param buyerPhone
     * @param buyerName
     * @return
     */
    public static PABillRegisterDataBuilder builder(String userid, String cardNo, String expMonth, String expYear, String buyerAuthNo, String cardPw, String buyerPhone, String buyerName) {
        return PABillRegisterData.builder().userid(userid).cardNo(cardNo).expMonth(expMonth).expYear(expYear).buyerAuthNo(buyerAuthNo).cardPw(cardPw).buyerPhone(buyerPhone).buyerName(buyerName);
    }

}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/OAPI/model/data/PABillRegisterData.swift
//
//  PABillRegisterData.swift
//  Payapp_iOS_Sample_V2
//
//  Created by 조상현 on 2022/09/26.
//

import ObjectMapper

class PABillRegisterData: Mappable {
 
    /**
     * cmd
     * 필수
     */
    var cmd:String?
    
    /**
     * 판매자 회원 아이디
     * 필수
     */
    var userid:String?

    /**
     * 카드번호
     * 필수
     */
    var cardNo:String?

    /**
     * 카드 유효기간(월)
     * 필수
     */
    var expMonth:String?
    
    /**
     * 카드 유효기간(년)
     * 필수
     */
    var expYear:String?

    /**
     * 구매자확인 개인:생년월일(YYMMDD) 6자리 , 사업자번호
     * 필수
     */
    var buyerAuthNo:String?
    
    /**
     * 카드 비밀번호 앞 두자리
     * 필수
     */
    var cardPw:String?

    /**
     * 구매자 전화번호
     * 필수
     */
    var buyerPhone:String?
    
    /**
     * 구매자 성함
     * 필수
     */
    var buyerName:String?

    /**
     * 구매자 아이디
     */
    var buyerId:String?
    
    required init?(map: Map){}
    init() {
    }
    
    func mapping(map: Map) {
        cmd                             <- map["cmd"]
        userid                          <- map["userid"]
        cardNo                          <- map["cardNo"]
        expMonth                        <- map["expMonth"]
        expYear                         <- map["expYear"]
        buyerAuthNo                     <- map["buyerAuthNo"]
        cardPw                          <- map["cardPw"]
        buyerPhone                      <- map["buyerPhone"]
        buyerName                       <- map["buyerName"]
        buyerId                         <- map["buyerId"]
    }
    
}
